data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -488,6 +488,75 @@ class ApiServerClient {
       });
   }
 
+  createConfigMapResource(configName, configParam) {
+    logger.info(`Creating ConfigMap ${configName} with data: ${configParam}`);
+    const metadata = {
+      name: configName
+    };
+    let data = {};
+    data = _.set(data, configParam.key, configParam.value);
+    const resourceBody = {
+      apiVersion: CONST.APISERVER.CONFIG_MAP.API_VERSION,
+      kind: CONST.APISERVER.CONFIG_MAP.RESOURCE_KIND,
+      metadata: metadata,
+      data: data
```

Comment:
remove `data` variable and directly put `_.pick(configParam, 'key', 'value')` here.

Reply:
Actually, @jagadish-kb, the idea is to have the resultant as `{keyName:valueName}`. I believe, `_.pick` will result in `{key: <keyName>, value: <valueName>}`. 
Hence, I modified this. What will be your suggestion?

Reply:
both approaches lead to the same end result, though with 2 lesser lines of code per my comment.